static/home/js/ligand_structural_search.js

```diff
@@ -0,0 +1,283 @@
+    setTimeout(function(){
+      toggleButtonClass('selection-button-ligand-by-id');
+      location.hash = '#';
+      window.location.href = url;
+    }, 500);
+
+  })
+  .fail(
+    function(jqXHR, textStatus, errorThrown){
+      if (jqXHR.hasOwnProperty('responseJSON')) {
+        if (jqXHR.responseJSON.hasOwnProperty('status_code')) {
+          if (jqXHR.responseJSON.status_code === 400) {
+            if (jqXHR.responseJSON.hasOwnProperty('msg')) {
+              location.hash = '#';
+              location.reload();
+              return;
+            }
+          }
+        }
+          
+      }
+      toggleButtonClass('selection-button-ligand-by-id');
+      showAlert("Something went wrong, please try again or contact us.", "danger");
+    });
+
+
+}
+
+
+
+
+
+
+function submitSelectionLigandBulkSearch(url) {
+  toggleButtonClass('selection-button');
+  let search_params_data = {
+    selection_ligand_chemical_structure_search_search_type_selection : $("#selection_ligand_chemical_search_type").val(),
+    search_text : $("#selection_ligand_bulk_search_textarea").val().trim(),
+    search_type : $("#selection_ligand_bulk_search_search_type").val(),
+    stereochemistry : $("#selection_ligand_bulk_search_stereochemistry").is(":checked"),
+    // limit : $("#ligand_structural_search_limit").val().trim()
+  }
+  
+
+  // set CSRF csrf_token
+  $.ajaxSetup({
+    headers:
+    { "X-CSRFToken": csrf_token }
+  });
+
+  // Submit search parameters
+  $.post("/ligand/read_input_ligand_bulk_search", search_params_data,  function (data) {
+    setTimeout(function(){
+      toggleButtonClass('selection-button');
+      window.location.href = url;
+    }, 500);
+  })
+  .fail(
+    function(jqXHR, textStatus, errorThrown){
+      if (jqXHR.hasOwnProperty('responseJSON')) {
+        if (jqXHR.responseJSON.hasOwnProperty('status_code')) {
+          if (jqXHR.responseJSON.status_code === 400) {
+            if (jqXHR.responseJSON.hasOwnProperty('msg')) {
+              location.hash = '#';
+              location.reload();
+              return;
+            }
+          }
+        }
+          
+      }
+      toggleButtonClass('selection-button');
+      showAlert("Something went wrong, please try again or contact us.", "danger");
+    });
+
+
+}
+
+function submitLigandStructuralSearch(url) {
+    let search_params_data = {
+      selection_ligand_chemical_structure_search_search_type_selection : $("#selection_ligand_chemical_search_type").val(),
+      smiles : $("#ligand_structural_search_smiles").val().trim(),
+      search_type : $("#selection_ligand_chemical_search_type").val(),
+      input_type : $("#ligand_structural_search_input_type").val(),
+      similarity_threshold : $("#ligand_structural_search_similarity_threshold").val().trim(),
+      stereochemistry : $("#ligand_structural_search_stereochemistry").is(":checked"),
+      // limit : $("#ligand_structural_search_limit").val().trim()
```

Comment:
same

Reply:
Solved in [bb9beb8](https://github.com/protwis/protwis/pull/550/commits/bb9beb8c3231460510e199f66a17a1c512c6c8e0).